examples/semi_supervised/plot_label_propagation_structure.py

```diff
@@ -28,12 +31,12 @@
 labels[0] = outer
 labels[-1] = inner
 
-# #############################################################################
+# %%
```

Comment:
Above, we should integrate the first figure that is shown below, i.e. the one showing the row data.